guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -93,25 +118,8 @@ Environment=GSS_USE_PROXY=1
 ----
 # systemctl restart httpd
 ----
-
-[IMPORTANT]
-====
-With direct AD integration, HBAC through {FreeIPA} is not available.
-As an alternative, you can use Group Policy Objects (GPO) that enable administrators to centrally manage policies in AD environments.
-To ensure correct GPO to PAM service mapping, add the following SSSD configuration to `/etc/sssd/sssd.conf`:
-
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-access_provider = ad
-ad_gpo_access_control = enforcing
-ad_gpo_map_service = +foreman
-----
-
-Here, _foreman_ is the PAM service name.
-ifdef::satellite[]
-For more information on GPOs, see {RHELDocsBaseURL}9/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#how-sssd-interprets-gpo-access-control-rules_applying-group-policy-object-access-control-in-rhel[How SSSD interprets GPO access control rules] in _{RHEL}{nbsp}9 Integrating RHEL systems directly with Windows Active Directory_ or {RHELDocsBaseURL}8/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#how-sssd-interprets-gpo-access-control-rules_applying-group-policy-object-access-control-in-rhel[How SSSD interprets GPO access control rules] in _{RHEL}{nbsp}8 Integrating RHEL systems directly with Windows Active Directory_.
-endif::[]
-====
+. Add a *Service Principal Name (SPN)* in your AD server to enable Kerberos connections.
```

Comment:
I've never had to do this since this command (from docs 5.3.3. Configuring Direct AD Integration with GSS-Proxy, step 6) adds Satellite as a client service - that is, adds it to AD and gets its keytab:
```
KRB5_KTNAME=FILE:/etc/httpd/conf/http.keytab net ads keytab add HTTP -U administrator -d3 -s /etc/net-keytab.conf
```

Reply:
Of course use of `administrator` may be questionable...